src/libtego/include/tego/tego.h

```diff
@@ -10,6 +12,9 @@ extern "C" {
 
 #define TEGO_TRUE  1
 #define TEGO_FALSE 0
```

Comment:
is there any reason this over <stdbool.h>?

Reply:
Makes it easier for future embedders that would need to write a marshalling layer between C and whatever else (python, java, etc) if libtego is used as a shared library. This explicitly defines the size of the bool and it's possible values all in the same header.